### PR TITLE
zh-trans: add content/case-studies/buffer/index.html

### PR DESCRIPTION
**Pull request by ten2ton**
zh-trans: add content/case-studies/buffer/index.html

### CODE REVIEW

File: content/zh/case-studies/buffer/index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+title: Buffer Case Study
```
Comment (sftim) [inline content/zh/case-studies/buffer/index.html]: `Buffer Case Study` is untranslated. Is that right?

Comment (ten2ton) [reply content/zh/case-studies/buffer/index.html]: Yes,I was negligent.
Thanks!

**Comment by markthink** [comment]
@ten2ton  需要解决一下冲突

**Pull request merged**